README.adoc

```diff
@@ -874,7 +874,7 @@ end
 # good
 RSpec.describe SomeJob do
   it 'updates user status' do
-    expect { SomeJob.perform_now(user) }.to change { user.status }.to(:updated) }
+    expect { described_class.perform_now(user) }.to change { user.status }.to(:updated) }
```

Comment:
There are a few mentions of `SomeJob` in the examples below as well.

I'm not sure, however, if this should be changed or not. There are two distinct styles, using `described_class` or not, it's a [controversial topic](https://github.com/rubocop-hq/rspec-style-guide/issues/104). Even though I try to take sides here, my preference slightly inclines towards explicitly specifying the class name. One reason is you don't have to look it up when you look at examples or send a snipped the code to someone. Another is that I don't really accept the argument that `described_class` reduces code churn when class is renamed: renaming job classes is a sensitive topic, described in another guideline here. Even though having to rename a job class doesn't alert people about the potential side effects of this, and I can't imagine a good spec that would prevent from it, my preference would still be to keep this additional portion of work when renaming.

Reply:
Thanks @pirj by the review. 
It makes sense. I didn't know that discussion about `described_class`, and I used it in all my projects. But the point that it could be changed in example groups (not only in the outermost describe, as I knew) makes it confusing.
Besides this, the possibility of this to be removed from RSpec4 is a good point to keep this guide as is.

Thank you a lot by the review and to point out that discussion, I ended up learning new things.

Reply:
> not only in the outermost describe, as I knew) makes it confusing

Indeed!

> to be removed from RSpec4

This is not yet decided, there is no roadmap for RSpec 4 I'm aware of. Just judging by maintainer comments in issues in RSpec repo.


You are welcome, and thank you for the first pull request sent to this guide (the fact that it's closed doesn't diminish the fact!) 🥇